tasks/typedoc.ts

```diff
@@ -0,0 +1,97 @@
+import ITask = grunt.task.ITask;
```

Comment:
Maybe we can submit an upstream fix for Grunt to not preface their interfaces with I ?